chimedb/core/mediawiki.py

```diff
@@ -0,0 +1,65 @@
+from .orm import base_model
+from .exceptions import ValidationError
+
+import hashlib
+import peewee as pw
+
+
+class MediaWikiUser(base_model):
+    """
+    Represents a MediaWiki user.
+
+    According to https://www.mediawiki.org/wiki/Manual:User_table
```

Comment:
Although the subset of the table we access is tiny, the table schema _can_ change between MediaWiki versions.  Can you add the latest known MediaWiki version for which this table model works with (which is probably the current one) so we have at least a flag that it's something we might want to re-check from time to time.

Reply:
Is this wrapping a MySQL VIEW of the wiki table? Or is this mirror users over? If the former, that should be noted, and I would suggest adding documentation which says how to create the view.